### ISSUE TITLE
[YSQL] Does yb-admin provide the function of clearing table cache?

### ISSUE THREAD

**Issue by suppersam1** [yugabyte/yugabyte-db#21299]
### Description

I inserted a large amount of data into a table and then established an index for a field. At this point, the cache of this field no longer needs to be used. The query now uses the cache of the index table. Can yb-admin provide a method to clean up the base table cache in this situation, without restarting the tserver.

### Issue Type

kind/question

### Warning: Please confirm that this issue does not contain any sensitive information

- [X] I confirm this issue does not contain any sensitive information.

**Comment by ddorian** [comment]
Hi @suppersam1 

The cache will be balanced automatically, no need to tell it.

**Comment by suppersam1** [comment]
> Hi @suppersam1
> 
> The cache will be balanced automatically, no need to tell it to be cleared if it won't be used by the app.

Will the allocated cache be automatically gc if it is not used for a long time?

**Comment by ddorian** [comment]
> Will the allocated cache be automatically gc if it is not used for a long time?

If some other table/index is being used, yes.

**Comment by suppersam1** [comment]
It means all tablets share one block_cache?

**Comment by suppersam1** [comment]
> > Will the allocated cache be automatically gc if it is not used for a long time?
> 
> If some other table/index is being used, yes.

It means all tablets share one block_cache?

**Comment by ddorian** [comment]
> It means all tablets share one block_cache?

Yes https://docs.yugabyte.com/preview/architecture/docdb/performance/#server-global-block-cache
